pkg/registry/generic/registry/store.go

```diff
@@ -106,7 +106,8 @@ type Store struct {
 	EnableGarbageCollection bool
 
 	// DeleteCollectionWorkers is the maximum number of workers in a single
-	// DeleteCollection call.
+	// DeleteCollection call.  Delete requests for the items in a collection
+	// are issued in parallel.
```

Comment:
I like having it stated. Intuitively, I can guess what this means, but I'd rather have it stated directly. Thanks